pulser/simulation/simulation.py

```diff
@@ -320,7 +321,8 @@ def make_xy_term() -> float:
                 )
             return xy
 
-        make_interaction_term = make_xy_term if self._interaction == 'XY' else make_vdw_term
+        make_interaction_term = make_xy_term\
+            if self._interaction == 'XY' else make_vdw_term
```

Comment:
Sorry for being a pain with these nits, but can you use `()` instead of `\` please?